wrappers/csharp/Utils.cs

```diff
@@ -11,7 +11,14 @@ public static byte[] StringToByteArray(string data)
                 return null;
             }
 
-            return Encoding.UTF8.GetBytes(data);
+            try
+            {
+                return Encoding.UTF8.GetBytes(data);
```

Comment:
I don't think we need this try-catch. The exceptions listed for GetBytes are `ArgumentNullException`, which we already check for, and `EncoderFallbackException`, but we do not set it `EncoderExceptionFallback`.

https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getbytes?view=netframework-4.8

Reply:
Fixed